### DISCUSSION TITLE
Custom Notifications

### DISCUSSION THREAD

**Discussion by imanimen** [mynaparrot/plugNmeet-server#201] [General]
Hi. How can I add some custom toast notification in the plugnmeet server and show to the client?

**Comment by jibon57** [comment]
Your question isn't clear to me.

**Reply by imanimen** [reply]
One example is like: when I click record, it says your meeting is now being recorded.
I want to add something when user reaches to the time limit of the meeting, show this to user """Your meeting is about to end in a minute"""

**Reply by jibon57** [reply]
OK, in that case you'll require to customized server side code & push data message to client.

**Reply by imanimen** [reply]
Can you please guide me how to implement that in to the server? Or should write it and build the code and dockerize it?
